packages/cli/src/lib/bundler/server.ts

```diff
@@ -61,12 +61,15 @@ export async function serveBundle(options: ServeOptions) {
         // See https://github.com/facebookincubator/create-react-app/issues/387.
         disableDotRule: true,
       },
-      https: url.protocol === 'https:',
+      https: false,
```

Comment:
I'd argue that serving the app as HTTPS is about as common as running it behind a reverse proxy in development :p

I think one way to go about this could be to add a `app.listen.https` config option that explicitly sets whether it should listen with https? That way you can have https in `app.baseUrl` but disable it with `app.listen.https: false`. What do you think?

Reply:
Agree, adding `app.listen.https` works.
I can add it to this PR or open a new PR for that.
Do you have a preference?

Reply:
👍 , I think this PR would be best, since it's all a bit intertwined

Reply:
I make the config changes. Please review.